aladdin.sh

```diff
@@ -28,6 +28,8 @@ PATH="$ALADDIN_BIN":"$PATH"
 DEFAULT_MINIKUBE_VM_DRIVER="virtualbox"
 DEFAULT_MINIKUBE_MEMORY=4096
 
+source "$SCRIPT_DIR/shared.sh"
```

Comment:
Moved the _extract_cluster_config_value function there and source it here

Reply:
nit: add a comment at the end of the source command `# to load _extract_cluster_config_value`

Reply:
done